Sdk/Track/Swrve.m

```diff
@@ -734,6 +809,10 @@ - (void)_deswizzlePushMethods
         SEL didFail = @selector(application:didFailToRegisterForRemoteNotificationsWithError:);
         [SwrveSwizzleHelper deswizzleMethod:didFail inClass:appDelegateClass originalImplementation:(IMP)didFailToRegisterForRemoteNotificationsWithErrorImpl];
         didFailToRegisterForRemoteNotificationsWithErrorImpl = NULL;
+        
+        SEL didReceive = @selector(application:didReceiveRemoteNotification:);
```

Comment:
It can indeed, but wouldn't it be too long? Wanted to have it short, maybe "didRecieveRemoteNotification" is a better name?